app/controllers/landing_controller.rb

```diff
@@ -0,0 +1,4 @@
+class LandingController < ApplicationController
```

Comment:
why did you add a LandingController when u already have the products controller?